docs/sources/configure-server/reference-configuration-parameters/index.md

```diff
@@ -53,7 +53,7 @@ Each variable reference is replaced at startup by the value of the environment v
 The replacement is case-sensitive and occurs before the YAML file is parsed.
 References to undefined variables are replaced by empty strings unless you specify a default value or custom error text.
 
-To specify a default value, use `${VAR:-default_value}`,
+To specify a default value, use `${VAR:default_value}`,
```

Comment:
This reverts https://github.com/grafana/pyroscope/pull/3055/files, perhaps you can change the template to preserve that change?

Reply:
Very nice catch. Fixed in https://github.com/grafana/pyroscope/pull/3123/commits/33dea843561abc2663d852a1dab2ae50486ad636